api/openapi-spec/swagger.json

```diff
@@ -17473,7 +17473,7 @@
       "type": "string"
```

Comment:
small nit: the "docs" in the commit message is not very accurate - kubevirt docs is a separate repo, more like "type description"

Reply:
I'd love to keep `docs:` in the commit title. If you prefer, I can change it to `chore:` since we're generating other files than just text files.